pandas/core/groupby/ops.py

```diff
@@ -578,13 +582,13 @@ def _cython_operation(
         return result, names
 
     def aggregate(
-        self, values, how: str, axis: int = 0, min_count: int = -1
+        self, values: np.ndarray, how: str, axis: int = 0, min_count: int = -1
```

Comment:
if we cant annotate _cython_operation values, then we cant do it here or in transform either